ui/Hotkeys.svelte

```diff
@@ -11,16 +10,37 @@
   };
 
   const onKeydown = event => {
+    const OSKey = navigator.platform.includes("Mac")
+      ? event.metaKey
+      : event.ctrlKey;
+
     if (event.target !== document.body) {
       return false;
     }
 
-    if (event.shiftKey && event.code === "KeyD") {
+    // To open help => OS modifier key + /
+    if (OSKey && event.code === "Slash") {
+      toggle(path.help());
+    }
+
+    // To open settings => OS modifier key + ,
+    if (OSKey && event.code === "Comma") {
+      toggle(path.settings());
+    }
+
+    // To open search => OS modifier key + p
+    if (OSKey && event.code === "KeyP") {
+      toggle(path.search());
+    }
+
+    // To open design system => OS modifier key + d
+    if (OSKey && event.code === "KeyD") {
       toggle(path.designSystemGuide());
     }
 
-    if (event.shiftKey && event.code === "Slash") {
-      toggle(path.help());
+    // To create a new project => OS modifier key + n
+    if (OSKey && event.code === "KeyN") {
```

Comment:
Shouldn't this be sensitive to the context? Feels a bit odd to jump to the project creation when you browse org members for example.

Reply:
Yeah that's how @brandonhaslegs had specced it in the issue. But I was thinking that right now the only thing you can really create is a new project. So I don't think it's a big deal to have this happen from anywhere, I can see in the future this opens a modal where you can choose what new thing you want to create:
- project
- org
- transfer
- etc

Reply:
Understood.